packages/bridge-ui/src/storage/service.ts

```diff
@@ -42,6 +42,11 @@ class StorageService implements Transactioner {
 
         const srcProvider = this.providerMap.get(tx.fromChainId);
 
+        // Ignore transactions from chains not supported by the bridge
+        if(!srcProvider) {
+          return null;
```

Comment:
i think this should actually just be `return`, which will return nothing from the `.map` and not create an entry in the array of `null`

Reply:
If we don't return anything using just `return` it will create an entry of `undefined` in the array. So we will still need to `.filter()` (which should answer this: https://github.com/taikoxyz/taiko-mono/pull/13153#discussion_r1106608946 as well).
![Screenshot 2023-02-15 at 9 08 46 AM](https://user-images.githubusercontent.com/108871478/218923145-74cc2f48-4054-4308-9dfe-60217a938328.png)

Reply:
Gotcha!